semgrep-core/src/cli/Main.ml

```diff
@@ -176,15 +176,6 @@ let mvars = ref ([] : Metavariable.mvar list)
 (*s: constant [[Main_semgrep_core.layer_file]] *)
 (*e: constant [[Main_semgrep_core.layer_file]] *)
 
-(*s: constant [[Main_semgrep_core.keys]] *)
```

Comment:
Very nice moving these functions out of Main!

Reply:
yep